source/images/introduction/perun_framework.svg

```diff
@@ -0,0 +1,875 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The downwards arrows appear tilted sideways when I view the picture.

(I don't know which line to comment, so I comment the first one.)

Reply:
Okay, i have changed it in [this](https://github.com/hyperledger-labs/perun-doc/pull/26/commits/1930b733d9013d4623987a6a1cbd26147691e39e) commit, now think it looks better.